src/wasm-binary.h

```diff
@@ -357,6 +367,18 @@ enum EncodedType {
   Empty = -0x40 // 0x40
 };
 
+enum EncodedHeapType {
+  func = -0x10,      // 0x70
+  extern_ = -0x11,   // 0x6f
+  any = -0x12,       // 0x6e
+  eq = -0x13,        // 0x6d
+  i31 = -0x17,       // 0x69
+  exn = -0x18,       // 0x68
+  signature = -0x20, // 0x60
+  struct_ = -0x21,   // 0x5f
+  array = -0x22      // 0x5e
```

Comment:
These are the values I found in the MVP document and elsewhere, but I'm not overly confident that all of them are correct.